docs/kyma/04-13-local-reinstallation-deprecated.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Reinstall Kyma (Scripts)
+type: Installation
+---
+
+>**WARNING:** Will be deprecated {TBD} with Kyma release 1.14
```

Comment:
```suggestion
>**WARNING:** Will be deprecated {TBD} with Kyma release 1.4
```
1.14? :D If we have this deprecation note, can we link to the new guide? Also, maybe we should inform in the installation guide that if somebody is using scripts, the guidelines are now in a separate doc.